meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/update-resin-supervisor

```diff
@@ -61,8 +61,8 @@ if [ -z "$API_ENDPOINT" -o -z "$CONFIG_PATH" ]; then
 fi
 
 # If the user api key exists we use it instead of the deviceApiKey as it means we haven't done the key exchange yet
-read DEVICE_API_KEY DEVICE_ID <<<$(jq -r '.apiKey // .deviceApiKey,.deviceId' $CONFIG_PATH)
-if [ -z "$DEVICE_API_KEY" -o -z "$DEVICE_ID" ]; then
+_device_api_key=${PROVISIONING_API_KEY:-DEVICE_API_KEY}
```

Comment:
Nope, this is for the case where an old device has had a host update and hasn't yet done a key exchange so it only has a "provisioning key" (which is really a provisioning+device key) eg if supervisor hasn't been updated to one that supports key exchange or hasn't had a chance to do it yet